src/handlers/ingress.ts

```diff
@@ -12,10 +12,25 @@ import handlers from './handlers'
 import { isAdmin } from '../util/permissions'
 import { unaliasInvocation } from '../util/unalias'
 import { handleEasterEgg } from './eggs/easter'
+import { addEntry } from '../util/cacheManager';
+import config from '../config/config'
 
 export async function handleMessage(msg: Message) {
-    // Get user or create one if it doesn't exist
     const authorId = msg.author.id
+    const channel = msg.channel.toString()
+
+    if(channel == config.channels.journals){
```

Comment:
For `addEntry` we actually need access to the `user` object since we need to look at their timezone. A good approach here could be that if the message doesn't come from journals, gratitude, or progress reporting, ignore it. Otherwise, look up the user who sent the message, and then pass it to `addEntry` if needed.